student.rb

```diff
@@ -0,0 +1,17 @@
+module StudentModule
+  require_relative 'person'
+  include PersonModule
```

Comment:
- [ ] Hello and nice job on the classes, but I do not know for what purpose this line `include PersonModule`. please remove it it's not necessary.